erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -590,6 +594,10 @@ def get_sales_orders(self):
 		so_filter += "and so.status = %(sales_order_status)s"
 
 	if self.item_code:
+		if frappe.db.exists('Item', self.item_code):
+			has_variant_bom = frappe.db.exists({'doctype': 'BOM', 'item': self.item_code})
+			if not has_variant_bom:
+				variant_of_bom = "'%s'" % frappe.db.get_value('Item', self.item_code, ['variant_of'])
```

Comment:
The variable stores the item code (either variant or template), make the variable describe that, and add a comment if you think it is still unclear. The current name is confusing
```suggestion
				bom_item = "'%s'" % frappe.db.get_value('Item', self.item_code, ['variant_of'])
```